unfetter-discover/unfetter-store
# Issue 1069 beta assess results

supports unfetter-discover/unfetter#1069

## problem
`/api/x-unfetter-assessments/<id>/assessed-objects` call returns 0 results for a capability assessment
_This is a partial fix for the result tab_

## changes
make the `assessed-objects` call capability aware
fix the `latest assessments` type

## test
* pull feature 
* create an assessment with a capability
* open the browser network tab
* visit the result tab on the beta assessment page
* verify a call like this; `/api/x-unfetter-assessments/x-unfetter-assessment--be380c63-9212-4ddc-aeab-37c7701eb475/assessed-objects`, returns data
